app/components/works/edit/attached_file_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr class="attached-files <%= 'dz-complete' if uploaded? %>" data-dropzone-target="preview">
+    <%= form.fields_for :attached_files, attached_file do |file_form| %>
+    <td class="file-level" style="--depth: <%= depth+1 %>rem" data-dropzone-target="fileName">
```

Comment:
Not a blocking concern, but a curiosity: might we be able to do the `+1` elsewhere? (like in the Ruby side of this component?)

Reply:
happy to move that `+1` to the component initialization